src/import/Import.mli

```diff
@@ -34,6 +34,10 @@ val ( $ ) : ('a -> unit) -> ('a -> 'b) -> 'a -> 'b
 (** Sequential composition of functions: [(f $ g) x] is exactly equivalent
     to [(f x) ; (g x)]. Left associative. *)
 
+exception Formatting_disabled
+    (** Raise this exception to Indicate that formatting has been disabled for the
+        current file *)
```

Comment:
Not for this PR, but looks like there is a separate existing issue with the indentation of docstrings on exception declarations.

Reply:
I noticed that too - I can add an issue for it if you'd like

Reply:
Fixed in 108a9d3388b21bd9668b80ae9bc9fcae35d513ce